src/main/target/common_fc_pre.h

```diff
@@ -102,6 +102,7 @@
 
 
 #define USE_CLI
+#define USE_GYRO_REGISTER_DUMP  // Adds gyroregisters command to cli to dump configured register values
```

Comment:
Considering the F3 flash space squeeze, I think this should be F4 only. If a developer needs it for testing they can always do a custom build.

Reply:
I'd like to leave it as is for initial testing but change to >= F4 before merge.  Reason being is that I still need testing for MPU9250 and I think there are only F3 targets with this gyro.  Want me to tag it as "don't merge" until them?

Reply:
If it builds for all targets for now then I'd rather add a comment like 'move to F4 only if needed' to the define. This way users wanting to test can use nightlies, and we'll have an easy win next time round we need to cull features on F3.